learning_journal_basic/security.py

```diff
@@ -0,0 +1,53 @@
+from pyramid.authentication import AuthTktAuthenticationPolicy
+from pyramid.authorization import ACLAuthorizationPolicy
+from pyramid.security import Allow
+from pyramid.security import Everyone, Authenticated
+from passlib.apps import custom_app_context as password_context
+import os
+
+
+class Root(object):
+    def __init__(self, request):
+        self.request = request
+
+    __acl__ = [
+        (Allow, Everyone, 'view'),
+        (Allow, Authenticated, 'root')
+    ]
+
+
+def verify_user(username, password):
+    is_user_authenticated = False
+    env_username = 'j'
+    env_password = '$6$rounds=637725$VjGuGRrTDTlXEAKK$dHlQHIwnJgIQ/qSmuerapBLFgznz6WWn.Cfq.ioMsMxbkUSQczYG5jXXiD2NsNjnfBx37Ej0/oP4YXy81aoNr.'
```

Comment:
You should be storing this value in your environment rather than putting it in code here.  That way you can have a different password in development than you have in production.